src/main/drivers/pwm_output.c

```diff
@@ -246,33 +226,39 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
 {
     memset(motors, 0, sizeof(motors));
     
-    uint32_t timerMhzCounter = 0;
     bool useUnsyncedPwm = motorConfig->useUnsyncedPwm;
 
+    uint16_t usMin = 0;
+    uint16_t usLen = 0;
+    uint16_t pwmRate = 500;
```

Comment:
it is independent if unsynced. This is the default rate to use when linked to looptime. i.e. the rate closest to the duty cycle at 100% but long enough to shut down future timing pulses - before the next pulse starts.